### PR TITLE
UTAPI-59 upgrade to Node 16

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

**Review by miniscruff — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e reset

File: lib/UtapiReindex.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ class UtapiReindex {
     _runScript(path, done) {
         const flags = this._buildFlags();
         this._requestLogger.debug(`launching subprocess ${path} with flags: ${flags}`);
-        const process = childProcess.spawn('python3.4', [
+        const process = childProcess.spawn('python3.7', [
```
Comment (nicolas2bert) [inline lib/UtapiReindex.js]: @tmacro is it ok to use `python3.7` here?
Now that we upgrade to Node 16, the base image uses Debian 10 that includes Python version 3.7

**Comment by nicolas2bert** [comment]
@bert-e wait

File: lib/UtapiReindex.js (hunk 1 above)
Comment (tmacro) [reply lib/UtapiReindex.js]: It should be, I regularly use 3.10 when I develop the script

**Review by tmacro — APPROVED** [review]

**Pull request merged**